application/backend/src/main/resources/apps/typerefinery/components/widgets/map/templates/flowsample.json

```diff
@@ -0,0 +1,14 @@
+{
+    "mapLat":"46.76336",
+    "mapLng":"-71.32453",
+    "markers":[{
+        "markerLat":"46.76186",
+        "markerLng":"-71.32247",
+        "popupText":"Marker 1"
+    },{
+        "markerLat":"46.76672",
+        "markerLng":"-71.32919",
+        "popupText":"Marker 1"
```

Comment:
Please provide an update to the popup text for Marker 2.

Even though this is sample data, make sure it looks good.

Reply:
Done